### ISSUE TITLE
[FEATURE] Can't referred to databricks_mlflow_experiment

### ISSUE THREAD

**Issue by holl59** [databrickslabs/terraform-provider-databricks#1144]
Hi there,

Actualy, it's impossible to refered to a existing experiment.

When i add it like a data bloc 

```hcl
data "databricks_mlflow_experiment" "test" {
  name          = "/Experiments/UC_test_oll"
}
```
I have the error : 
```hcl
│ The provider databrickslabs/databricks does not support data source "databricks_mlflow_experiment".
│
│ Did you intend to use the managed resource type "databricks_mlflow_experiment"? If so, declare this using a "resource" block instead of a 
│ "data" block.
```
So I add it like a ressource and went to import it :
```hcl
terraform import module.conf_dbk_lab.databricks_mlflow_experiment.test /Experiments/UC_test_oll
```

I have :
**module.conf_dbk_lab.databricks_mlflow_experiment.test: Importing from ID "/Experiments/UC_test_oll" ...**
and the import do nothing ... i have to kill the process

### Configuration
```hcl
    databricks = {
      source = "databrickslabs/databricks"
      version = "0.5.1"
```

Thanks

**Comment by nfx** [comment]
@holl59 `databricks_mlflow_experiment` is imported by experiment id (number), not the path. also killing the TF process is never a good idea :)

**Comment by nfx** [comment]
also - PRs are welcome. there was a recent similar feature added: https://github.com/databrickslabs/terraform-provider-databricks/commit/67dea81f0e5444211ded4ae598476e58424440ce / https://github.com/databrickslabs/terraform-provider-databricks/blob/master/jobs/data_jobs.go

**Comment by holl59** [comment]
I just create a PR to update documentation #1148

**Comment by nfx** [comment]
Closed in #1148
